### ISSUE TITLE
Iron is too dominant, add tin and bronze? [SUGGESTION]

### ISSUE THREAD

**Issue by Ketrai** [StrangeLoopGames/EcoSuggestions#1336]
I've noticed together with some other folk that iron is a bit too important, especially early game. Iron tools, as well as bakery ovens, iron stoves, and some other T2 equipment  are all made out of it. This gives iron producers a huge benefit, and can bottleneck communities if there isn't enough available. It also makes it pointless for miners to set up copper mines before trucks or proper roads are made to expand elsewhere.

Suggestion
- Add tin. We already have Copper, this would encourage collaboration between miners and smelter.
- Add bronze. This would be the alternative to iron, and perhaps be needed for certain progression items that currently only need iron.

**Comment by PsychoRomeo** [comment]
I don't see this solving the problem.  A solution in my mind would adjust iron's need, not attempt to dilute it with a new alloy.

**Comment by Ketrai** [comment]
Hewnlogs has mortared stone as alternative.
Bricks has lumber and glass as alternative.
Baking and cooking complement each other.

If anything, having only one real product in an industry will always cause supply issues. Especially if they're dependent on the same skill.

**Comment by PsychoRomeo** [comment]
Mortared stone is not an alternative to hewn logs.  There are a billion recipes that accept hewn logs and not mortared stone, or don't have a mortared stone counterpart.  Pretty much the only place the two are remotely equal is when it comes to buildings and furniture.  Same with lumber/bricks, and baking/cooking's relationship isn't at all comparable to your iron problem.  

If the bottleneck is iron bars, produced by a smelter, then adding bronze bars using the same bottleneck does not resolve the bottleneck in the slightest.  It would be like adding bricks as an alternative t2 building and furniture material to lumber, but had them also produced at the sawmill by a carpenter.

**Comment by Jacqhammer** [comment]
I tend to agree with the idea that mortared stone and hewn logs don't share the same relationship. But I do think bronze would be a great addition to early game development. In the game copper is not really used until end game, but in reality, copper was the first metal man smelted. Adding copper in early game functions like cart wheels, door hinges and such would increase smelting beyond iron farming. 

It would make logical sense to put it in the game early on. Along with allowing copper/crushed copper to spawn in grasslands, desert and forests at the same level as dirt. but keeping the large veins deep underground in the cold forests.

**Comment by danisani1** [comment]
iron is much harder to smelt then copper. I think it just would be a cool and more deepening addition. There could be a 'metal' tag for tools and some other items. It could make smelting more interesting (all the fun is in mining and even that would be more interesting then)
